### ISSUE TITLE
Typing continues link format

### ISSUE THREAD

**Issue by leo-zhao** [quilljs/quill#211]
When inserting a hyperlink there is a problem in that unless you have put a space after the display text used for the link, you are unable to restart writing again as all new text gets automatically attached to the display text and becomes part of the hyperlink. This means you have to complete whole sentences and then go back and select the text that you want to link from. You can’t create the hyperlink as you go.

The only way other way around this is to hit enter and start a new line as this breaks the hyperlink at the end of the old line.

**Comment by rolandobloom** [comment]
This bug is still present in the latest quill version. Is it going to be fixed or at least included in the TODO list?

**Comment by rolandobloom** [comment]
This is not related to
[Cannot use select formats in IE8 #59](https://github.com/quilljs/quill/issues/59)

In Mozilla and IE it is not possible to turn off linking option unless you hit Enter and start a new line.
After some debugging i discovered that in Chrome 'carriage return' is added by default, while in Mozilla and IE it is not.

This is what ops object looks like in Chrome:
```javascript
0:
  attributes:
                   insert: "aaa"
1: 
  insert: "↵"
```
while in Mozilla and IE
```javascript
0:
  attributes:
                   insert: "aaa"
1: 
  insert: ""
```

**Comment by rolandobloom** [comment]
Maybe someone is aware of the cause of this problem and could point me in the right direction?
This bug occurs only in Mozilla and IE (i didn't check it on mac).

**Comment by jhchen** [comment]
The cause seems to be it's just the default browser behavior so the solution would involve detecting and overwriting this.

**Comment by jhchen** [comment]
This seems to have been fixed in the 1.0 beta release. If you place your cursor after a link and continue typing in FF, the link will continue. However you are able to place your cursor after a link, click the link button and typing will not continue the link format.
